game/script-topics.rpy

```diff
@@ -11550,3 +11550,154 @@ label talk_enable_no_topics_reminder:
+    if already_discussed_topic:
+        n 7csrca "..."
+        n 3fcsflesi "..."
+        n 3cllfl "You know,{w=0.2} [player]..."
+        n 7tnmsl "I'm pretty sure I went through it all before.{w=0.75}{nw}"
+        extend 7clrflsbl " And I don't even know why it still bothers me so much."
+        n 4fsrem "But I {i}still{/i} can't stand when some people feel the need to just blast what {i}they{/i} wanna listen to out loud in front of everyone else."
+        n 2fsrca "..."
+        n 2fnmfll "What?{w=0.75}{nw}"
+        extend 2csqfll " Do I look like I'm making it all up or something?{w=0.75}{nw}"
+        extend 4fcsgsl " It's the {i}worst{/i}!"
+
+    else:
+        n 7cslsl "..."
+        n 3fslan "Tch!"
+        n 3fsrbo "..."
+        n 3ccsss "Heh."
+        n 4cllfl "Hey,{w=0.2} [player]...{w=1}{nw}"
+        extend 2cnmaj " you wanna know what winds me up?"
+        n 2fnmfl "And I mean{w=0.5}{nw}"
+        extend 4fsqem " {i}really{/i}{w=0.5}{nw}"
+        extend 4fsran " gets on my nerves?"
+        n 2fcsemesi "..."
+        n 2fllaj "When you're taking some sort of public transport,{w=0.2} or you're just hanging around somewhere.{w=0.75}{nw}"
+        extend 2fllfl " That kind of thing."
+        n 4fcsanean "...And then some total jerk feels the need to pull our their phone just to blast out what {i}they{/i} feel like listening to."
+        n 4fsrsl "..."
+        n 2fnmwrl "H-{w=0.2}hey!{w=0.75}{nw}"
+        extend 2fcspol " I'm being serious here,{w=0.2} [player]!{w=0.75}{nw}"
+        extend 4fbkwrl " It's the {i}worst{/i}!"
+
+    n 3flrem "I mean,{w=0.2} come on.{w=0.75}{nw}"
+    extend 3fupwr " Where do I even {i}start{/i}?"
+    n 6fcsgs "First off,{w=0.5}{nw}"
+    extend 3fllan " you can basically guarantee that whatever it is,{w=0.75}{nw}" 
+    extend 3fslan " it's {i}never{/i} gonna sound good."
+    n 4fbkwr "Nothing ever {i}does{/i} when you're blasting it out of crappy phone speakers or busted-up headphones!"
+    n 1nsqem "And{w=0.3} Oh.{w=0.75}{nw}"
+    extend 2csqfl " My.{w=0.75}{nw}"
+    extend 2fsqem " God,{w=0.5}{nw}"
+    extend 4fsrfu " [player]."
+    n 4fcsup "{i}The echoing{/i}."
+    n 3fsqgs "Do you even know how much worse it gets when you're stuck inside the train,{w=0.5}{nw}" 
+    extend 3fcsgs " or literally anything else?"
+    n 3fcsan "Talk about an earache.{w=0.75}{nw}"
+    extend 7fllem " Especially if you just wanted to relax before the day starts!"
+    n 4fsrem "{i}And{/i} you can forget about trying to hold any kind of conversation too with all that noise in the background."
+    n 2fcssl "Heh.{w=0.75}{nw}"
+    extend 2fsqfr " Then you know what the worst part is,{w=0.2} [player]?"
+    n 4fllem "If you even try to call them out,{w=0.5}{nw}"
+    extend 4csqem " you just know they'll make a massive scene over it too,{w=0.5}{nw}" 
+    extend 4fslfu " or make it sound like you're just freaking out!"
```

Comment:
It might be nice, but not necessary to add the aspect of parents giving their children tablets or phones to keep them occupied, but don't give them headphones or make them mute their stupid ringing apps!

Reply:
Sure, I can add a line for that. This is already quite long for a wind-up, so it'll be kept short